tests/appsec/test_asm_standalone.py

```diff
@@ -56,6 +59,40 @@ def _assert_tags_value(span, obj, expected_tags):
         except (KeyError, AssertionError) as e:
             return False
 
+    def setup_check_product_is_enabled(func):
```

Comment:
Same comment as below, I think it'll be easier to understand for non python user if we use a simple method called directly in `setup` bodies.